composer.lock

```diff
@@ -2424,7 +2424,7 @@
         },
         {
             "name": "robmorgan/phinx",
-            "version": "0.10.8",
+            "version": "0.11.1",
```

Comment:
this is not the way to update a `composer.lock` file. You need to run the command `composer update robmorgan/phinx` to only update the phinx dependency